### PR TITLE
Feature/base structure for main component

### PR DESCRIPTION
**Pull request by NIKaragu**
Created a base structure for Main component and also subcomponents for Main + made base styles for Main component.
Also was fixed structure of MainPage and its styles

### CODE REVIEW

File: src/components/Main/NewModelsList/NewModelList.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+          <button className="new-models-slider__button new-models-slider__button--prev"></button>
+        </div>
+      </div>
+
+      <ul className="new-models-slider__list">
+        {/* TODO: at this stage we need to load phone cards from server, map given list and render it */}
+        <NewModelItem />
+      </ul>
+    </div>
```
Comment (k-marchuk) [inline src/components/Main/NewModelsList/NewModelList.tsx]: There are 2 components with such sliders on Main page (hot prices and brand new models), and one more on Item card page (you may also like). They all look identical, only content and headers differ. Maybe it makes sense to create only one component, something like ProductsSlider and reuse it?
More thoughts are needed :thinking:

**Review by k-marchuk — CHANGES_REQUESTED** [review]
Very nice work!
I left one suggestion.

File: src/components/Main/NewModelsList/NewModelList.tsx (hunk 1 above)
Comment (NIKaragu) [reply src/components/Main/NewModelsList/NewModelList.tsx]: Good idea, done)

**Comment by k-marchuk** [comment]
It is seems that you are working on implementing the Home Page, but we have a separate task for that in Step 5 https://app.clickup.com/t/86c0a7fx3. 
It's seems we have to clarify with Danylo what is main component.

**Comment by NIKaragu** [comment]
> It is seems that you are working on implementing the Home Page, but we have a separate task for that in Step 5 https://app.clickup.com/t/86c0a7fx3. It's seems we have to clarify with Danylo what is main component.

Our MainPage is equal to HomePage, we can rename it later or adjust some moments according to next tasks. 
Just let's move on step by step with our current tasks, don't run ahead so far)

**Pull request merged**